src/runtime/uniboot.h

```diff
@@ -61,14 +59,22 @@ extern void * AP_BOOT_PAGE;
 
 /* runloop timer minimum and maximum */
 #define RUNLOOP_TIMER_MAX_PERIOD_US     100000
-#define RUNLOOP_TIMER_MIN_PERIOD_US     10
```

Comment:
Yeah, this is a little sad. Basically if we didn't have a lower bound, then a unix timer could be registered for a 1ns interval - which would indeed fire immediately. If said timer is made to be periodic, we'll just get stuck endlessly serving timer interrupts as soon as we do the frame return to userspace (this happened in the time runtime test - which does test 1ns intervals - if for no other reason than they should work without hanging up the kernel).

The upper bound could be taken out - I think some time ago I was thinking it would act like an HZ timer, forcing scheduling to happen if things got stuck as a result of a bug...but that's kinda lame - plus I think we'd want to know if such a condition occurred, and it would just go unnoticed right now. (Plus lwIP has its own periodic timers that run, so it's not like the timer heap would ever truly stay idle...)

I think we should revisit our general strategy with respect to timers...knowing how expensive it can be to program them...maybe we could accept more slop in exchange for throughput / less vm exits.